Add support for arm64 architecture

**Describe the feature**

Since the DockerHub image is not compatible with ARM, I tried to build the project myself on my rPi 4b running Ubuntu 20.04.2. The build failed during the testing phase with the message

> ?       mittens/pkg/warmup      [no test files]
> FAIL    mittens/test [build failed]
> FAIL
> make: *** [Makefile:7: test] Error 2

It would be great if the project was converted to multi-architecture or adapted to build on arm machines.

**Rational**

Mittens seems to be unique and I would really like to use it to warm pods on my Raspberry Pi Kubernetes cluster so that they can be brought into load fully warmed,  when the HPA scales out, since it does this under load and adding slower pods is less desirable. But I'm sure I am preaching to the converted!

--- a/.github/workflows/docker-build-push.yml
+++ b/.github/workflows/docker-build-push.yml
@@ -22,6 +22,12 @@ jobs:
           tags: |
             type=ref,event=tag
 
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
       - name: Login to DockerHub
         uses: docker/login-action@v1 
         with:
@@ -31,7 +37,7 @@ jobs:
       - name: Build and push
         uses: docker/build-push-action@v2
         with:
+          platforms: linux/amd64,linux/arm64,linux/arm/v7
           push: true
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
-          

